operators/multiclusterobservability/controllers/placementrule/placementrule_controller.go

```diff
@@ -578,9 +578,9 @@ func createAllRelatedRes(
 			continue
 		}
 
-		if err := deleteObsAddon(ctx, c, ep.Namespace); err != nil {
-			allErrors = append(allErrors, fmt.Errorf("failed to deleteObsAddon: %w", err))
-			log.Error(err, "Failed to delete observabilityaddon", "namespace", ep.Namespace)
+		if err := deleteManagedClusterRes(c, ep.Namespace); err != nil {
```

Comment:
@dbuchanaRH I think we should probably call both. First deleteObsAddon() followed by deleteManagedCllusterRes(). The later won't delete/cleanup the Observability addon I believe.